apidoc/Annotation.yml

```diff
@@ -76,6 +76,131 @@ properties:
     type: Titanium.UI.View
     since: "3.1.0"
 
+  - name: showMarker
```

Comment:
We may want to call this `showAsMarker`, here is why: Developers could misunderstand this property to actually "show" an annotation when enabling it. What do you think?

Reply:
EDIT: Looking at all the properties, they all have the `marker` prefix on them, which makes the property names pretty long. What do you think about the following: We add a `marker` property that is an object of keys. Example:
```js
Map.createAnnotation({
  latitude: xxxx,
  longitude: xxxx,
  marker: {
    glyphTextColor: 'red',
    ...
  }
});
```
Using that, we won't need a toggle property to indicate the usage, we would just check the property against nil and pass over the properties if set. That also has the big advantage that we don't need to call `setNeedsRefreshingWithSelection` for every property-change, but can do it once when marker-properties are set.

Reply:
"showAsMarker" is good idea. Made changes for same. But if we make our marker api as you mentioned -
Map.createAnnotation({
  latitude: xxxx,
  longitude: xxxx,
  marker: {
    glyphTextColor: 'red',
    ...
  }
});
If developer want to set the glyphTextColor (e.g) at run , it will not be convenient. It is not necessary to set all marker properties at creation only. And there are certain conditions,  only after that refreshing of annotations happen e.g (delegate!=nil && [delegate viewAttached]). So I think current implementation is fine. What you think ?